fronts-client/src/components/form/ArticleMetaForm.tsx

```diff
@@ -763,45 +764,91 @@ class FormComponent extends React.Component<Props, FormComponentState> {
 											}
 										/>
 									</InputGroup>
+									{primaryImage &&
+										!!primaryImage.src &&
+										!this.props.showMainVideo &&
+										!this.props.imageSlideshowReplace && (
+											<InputGroup>
+												<ConditionalField
+													permittedFields={editableFields}
+													name="imageReplace"
+													component={InputCheckboxToggleInline}
+													label="Use replacement image"
+													id={getInputId(cardId, 'image-replace')}
+													type="checkbox"
+													default={false}
+													onChange={() => this.changeImageField('imageReplace')}
+												/>
+											</InputGroup>
+										)}
+								</ToggleCol>
+								<Col flex={2}>
+									<InputLabel htmlFor="media-select">Select Media</InputLabel>
 									<InputGroup>
-										<ConditionalField
-											permittedFields={editableFields}
-											name="showMainVideo"
-											component={InputCheckboxToggleInline}
-											label="Show video"
-											id={getInputId(cardId, 'show-video')}
-											type="checkbox"
-											onChange={() => this.changeImageField('showMainVideo')}
+										<Field
+											component={InputRadio}
+											disabled={
+												editableFields.indexOf(this.getImageFieldName()) === -1
+											}
```

Comment:
Previously unusable buttons were hidden; now we prefer to disable them. We could in future include a label explaining why the button is disabled.